Library/Homebrew/requirements/xorg_requirement.rb

```diff
@@ -0,0 +1,24 @@
+require "requirement"
+
+class XorgRequirement < Requirement
+  fatal true
+  default_formula "linuxbrew/xorg/xorg"
+
+  env { ENV.x11 }
```

Comment:
Do we want this line? I'm thinking not. See https://github.com/Linuxbrew/brew/blob/master/Library/Homebrew/extend/ENV/std.rb#L262-L263
```ruby
  def x11
    return unless OS.mac?
```